components/transactions/TransactionItem.js

```diff
@@ -154,6 +156,36 @@ const TransactionItem = ({ displayActions, collective, transaction, onMutationSu
 
   const displayedAmount = getDisplayedAmount(transaction, collective);
 
+  const transactionDetailLink = transactionKind => {
+    return (
+      <StyledButton
+        data-cy={transactionKind === TransactionKind.EXPENSE ? 'expense-details' : 'transaction-details'}
```

Comment:
Why do we need this?

Reply:
Seems an error on my part, we don't need this and I've removed the `if` condition.